templates/index.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %} 
-{% load static %} 
+{% load i18n %}
```

Comment:
Missing trans tags under on 
- `labels` https://github.com/hikaya-io/activity/blob/30508e514135d1fa971f5356cce5cc873a0f0428/templates/index.html#L371-L376
- `Status` https://github.com/hikaya-io/activity/blob/30508e514135d1fa971f5356cce5cc873a0f0428/templates/index.html#L386
- https://github.com/hikaya-io/activity/blob/30508e514135d1fa971f5356cce5cc873a0f0428/templates/index.html#L412
- https://github.com/hikaya-io/activity/blob/30508e514135d1fa971f5356cce5cc873a0f0428/templates/index.html#L417
- https://github.com/hikaya-io/activity/blob/30508e514135d1fa971f5356cce5cc873a0f0428/templates/index.html#L430

Reply:
I left all translation related to JavaScript or Python code for another time. This also include strings returned by Django, such as the Django notifications.